spec/support/page_objects/sections/location_form.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module PageObjects
+  module Sections
+    class LocationForm < PageObjects::Sections::Base
```

Comment:
I think this should be named `SchoolForm` now

Reply:
Yep

Reply:
Deleted